packages/testing/src/playwright/bypassBotProtection.ts

```diff
@@ -0,0 +1,33 @@
+import type { Page } from '@playwright/test';
+
+type BypassBotProtectionParams = {
+  page: Page;
+  options?: {
+    /*
+     * The frontend API URL for your Clerk dev instance, without the protocol.
+     * If provided, it overrides the Frontend API URL parsed from the publishable key.
+     * Example: 'relieved-chamois-66.accounts.dev'
+     */
+    frontendApiUrl?: string;
+  };
+};
+
+const TESTING_TOKEN_PARAM = '__clerk_testing_token';
+
+export const clerkBypassBotProtection = async ({ page, options }: BypassBotProtectionParams) => {
+  const fapiUrl = options?.frontendApiUrl || process.env.CLERK_FAPI;
```

Comment:
🔧 We should probably throw an error if we're missing the frontend API url, or at least make this helper a noop.